### ISSUE TITLE
What path dropout probability has been adopted in this work on CIFAR-10?

### ISSUE THREAD

**Issue by Windaway** [lightaime/sgas#6]
0.2 or 0.3?
Paper 0.3 Code 0.2?

**Comment by lightaime** [comment]
As we mentioned in the paper, we used the same parameters as DARTS.  I double check the original DARTS paper. They use path dropout probability as 0.2. So it seems to be a typo in our paper. We will fix it later. Please follow the released code and use path dropout probability as 0.2. Sorry for the inconvenience. Let me know if you have any questions.

**Comment by Windaway** [comment]
PC-DARTS/P-DARTS use 0.3.
Will the learning rate decay and batchsize affect the performance on Imagenet.
Linear vs Cosine? Batchsize 1024 vs 128?

**Comment by lightaime** [comment]
As for the training on ImageNet, we follow the same setting as P-DARTS. Ideally, the learning rate decay and batch size should not effect that much. But in our experiments, we only trained the models with Linear Decay and a batch size 1024 on 8 GPUs.

**Comment by Windaway** [comment]
If you follow P-DARTS you should use linear decay(as the code)?  Use cosine annealing?

**Comment by lightaime** [comment]
Yes, it is Linear Decay. Please see the code for those training details.

**Comment by lightaime** [comment]
Sorry for the confusion. To clarify, we don’t tune any parameters or training strategies. For the CIFAR10 experiments, we use the same setting as DARTS. For the ImageNet experiments, in order to train faster, we follow the same setting that how P-DARTS trained on 8 GPUs.
